### ISSUE TITLE
ld: warning: meta method 'UIDatePickerMode:' in category

### ISSUE THREAD

**Issue by JeremiahUL** [henninghall/react-native-date-picker#214]
**Describe the bug**
A warning on the latest 3.0.2 where the meta method 'UIDatePickerMode' is duplicated exactly like the React-Native Core 'UIDatePickerMode', this causes a warning that I can seem to fix or suppress.
Is there a way to change the name of the method so it doesn't clash with that one.
Is there a way to suppress warnings like this

**Expected behavior**
No warnings ... No method duplications

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
export default class App extends Component {

  state = { date: new Date() }

  render = () =>
    <DatePicker
      date={this.state.date}
      onDateChange={date => this.setState({ date })}
    />

}
```

**Smartphone (please complete the following information):**
 - OS: [iOS]
 - React Native version [e.g. 0.62.2]
 - react-native-date-picker version [e.g. 3.0.2]

**Comment by henninghall** [comment]
Which version of Xcode and iOS are you using?

**Comment by JeremiahUL** [comment]
Hi @henninghall , XCode version 11.6

**Comment by JeremiahUL** [comment]
Hi were you able to see or replicate the warning.

**Comment by JeremiahUL** [comment]
Hi @henninghall @hungneox @davegurnell @grifotv 
Please I am sorry for this but can I fix this in anyway. Even if its a quick fix. Please help

**Comment by henninghall** [comment]
I have replicated the warning but it doesn't seem to affect the usage. Do you mean that you have issues compiling or using the picker?

**Comment by JeremiahUL** [comment]
I am not having issue compiling or anything like that. It is totally fine in other projects but this particular project does automatic deployment and fails on every warning. If I were able to suppress it, I would have

**Comment by JeremiahUL** [comment]
Wow. This is awesome

**Comment by henninghall** [comment]
Fixed in v3.1.0
